core/shared/src/test/scala/com/softwaremill/sttp/testing/HttpTest.scala

```diff
@@ -343,47 +359,45 @@ trait HttpTest[R[_]]
       }
     }
 
-    "keep a single history entry of redirect responses" in {
-      r3.send().toFuture().map { resp =>
-        resp.code should be(200)
-        resp.unsafeBody should be(r4response)
-        resp.history should have size (1)
-        resp.history(0).code should be(302)
+    if (usesFollowRedirectsBackend) {
```

Comment:
maybe we could move these tests to `HttpTestExtensions`, like in the other cases for tests which only work on the jvm, no in the js environment?

Reply:
They would be relevant to any `nodejs` backend so I don't know if its worth moving them?

Reply:
Ah, so there might be other JS-based backends?

Reply:
Someone could write another browser one based on XHR or one written in nodejs (nodejs would allow access to the response). Probably won't happen so I'll move them into `HttpTestExtensions`.